src/main/java/Event.java

```diff
@@ -0,0 +1,29 @@
+import java.time.LocalDateTime;
+
+/**
+ * Represents a Task with a time of event attached
+ */
+
+public class Event extends Task {
+    private String time;
+    private LocalDateTime localDateTime;
+
+    public Event(String name, boolean isDone, String time, LocalDateTime localDateTime) {
+        super(name, isDone);
+        this.time = time;
```

Comment:
seems that the "time" field and the "getTime" method is only used in the tests...is it necessary to include them?